phoenix-pherf/src/main/java/org/apache/phoenix/pherf/workload/MultithreadedDiffer.java

```diff
@@ -51,7 +53,7 @@ private void diffQuery() throws Exception {
         Date startDate = Calendar.getInstance().getTime();
         String newCSV = queryVerifier.exportCSV(query);
         boolean verifyResult = queryVerifier.doDiff(query, newCSV);
-        String explainPlan = queryVerifier.getExplainPlan(query);
```

Comment:
Could you add the explainPlan stuff as a different Jira/CL? It might get checked in faster if it's detached from this CL. This one might need a little more love.